README.md

```diff
@@ -13,19 +13,17 @@
 
 # Overview
 
-This is a CLI is primarily for operators of the holograph network. The CLI includes tools to move NFTs between networks and view their status.
+The Holograph CLI enables you to interact with Holograph Protocol, in addition to participating as an operator in the Operator Network.
 
 <!-- toc -->
-
-- [Overview](#overview)
-- [Usage](#usage)
-- [Commands](#commands)
+* [Overview](#overview)
```

Comment:
I think your editor might be updating these automatically. If you run `yarn prettier:fix` it should standardize everything